priv/repo/migrations/20180412044053_create_votes.exs

```diff
@@ -0,0 +1,18 @@
+defmodule ShowNTell.Repo.Migrations.CreateVotes do
+  use Ecto.Migration
+
+  def change do
+    create table(:votes, primary_key: false ) do
+      add :value, :integer
+      add :talk_id, references(:talks, on_delete: :nothing)
+      add :user_id, references(:users, on_delete: :nothing)
+
+      timestamps()
+    end
+
+    create unique_index(:votes, [:user_id, :talk_id], name: :votes_talk_user_id_index)
```

Comment:
After thinking about this constraint, I wonder if we should reconsider it in the future. This makes it so that you would never be able to change your vote once you made it. We might want to modify this to allow the user to +1 or -1, but not +2 or -2. I think we can deal with this detail later though.